community/demos.md

```diff
@@ -2,6 +2,12 @@ If you have a demo of CloudEvents in action, please add a link here. If there
 isn't an associated blog or github repo, feel free to add descriptive text as a
 markdown file in `community/demos/`.
 
+- [Orchestrating Cloud Events](https://salaboy.com/2020/05/18/orchestrating-cloud-events-with-zeebe/)
+
+  - Description: Blog post describing how Cloud Events can be orchestrated using an orchestration engine. Example included that you can run in your own environment with links to GitHub repositories needed.
+  - Instructions: You can find the [Demo Code and instructions here](https://github.com/salaboy/zeebe-cloud-events-examples).
+  - Tags: Cloud Events, Java, Orchestration, Workflow, KNative, Example
```

Comment:
Why this line? None of the other ones have this

Reply:
line removed to avoid a mismatch with other entries